Cargo.lock

```diff
@@ -2785,8 +2789,7 @@ dependencies = [
 [[package]]
 name = "pkcs5"
 version = "0.7.1"
-source = "registry+https://github.com/rust-lang/crates.io-index"
-checksum = "e847e2c91a18bfa887dd028ec33f2fe6f25db77db3619024764914affe8b69a6"
+source = "git+https://github.com/bitwarden/rustcrypto-formats.git?rev=2b27c63034217dd126bbf5ed874da51b84f8c705#2b27c63034217dd126bbf5ed874da51b84f8c705"
```

Comment:
@quexten Why did this only get updated in this PR? It seems related to https://github.com/bitwarden/sdk-internal/pull/150

Reply:
Merging main should make this diff disappear since we now always patch pkcs5: https://github.com/bitwarden/sdk-internal/pull/211